lib/src/components/NxTransferListHalf/__tests__/NxTransferListHalf.test.tsx

```diff
@@ -31,6 +31,8 @@ describe('NxTransferListHalf', function() {
 
   beforeEach(mockTransferListLayout);
 
+  beforeEach(mockTransferListLayout);
```

Comment:
Bad merge it seems, duplicates the line above.

Reply:
> Bad merge it seems, duplicates the line above.

good catch, done at 0f4f473fa1c702f621a9e682c53d40d0213b22d2